Content.Server/FeedbackSystem/FeedbackCommand.cs

```diff
@@ -0,0 +1,66 @@
+using Content.Server.Administration;
+using Content.Shared.Administration;
+using Content.Shared.FeedbackSystem;
+using Robust.Shared.Console;
+using Robust.Shared.Player;
+using Robust.Shared.Prototypes;
+
+namespace Content.Server.FeedbackSystem;
+
+[AdminCommand(AdminFlags.Server)]
+public sealed class FeedbackPopupCommand : LocalizedEntityCommands
+{
+    [Dependency] private readonly IPrototypeManager _proto = default!;
+    [Dependency] private readonly SharedFeedbackSystem _feedback = default!;
+
+    public override string Command => Loc.GetString("feedbackpopup-command-name");
+
+    public override void Execute(IConsoleShell shell, string argStr, string[] args)
+    {
+        if (args.Length != 2)
+        {
+            shell.WriteError(Loc.GetString("feedbackpopup-command-error-wrong-arguments"));
+            return;
+        }
+
+        if (!int.TryParse(args[0], out var entityUidInt))
+        {
+            shell.WriteError(Loc.GetString("feedbackpopup-command-error-invalid-uid"));
+            return;
+        }
+
+        var netEnt = new NetEntity(entityUidInt);
+
+        if (!EntityManager.TryGetEntity(netEnt, out var target))
+        {
+            shell.WriteLine(Loc.GetString("feedbackpopup-command-error-entity-not-found"));
```

Comment:
```suggestion
            shell.WriteLine(Loc.GetString("shell-could-not-find-entity-with-uid", ("uid", entityUidInt)));
```

Rest of the loc strings I think are good. (I'm sus on the existing invalid prototype one.)